doc/DeveloperGuide.md

```diff
@@ -94,6 +94,8 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should respond to all commands within two seconds under normal load.
```

Comment:
What is normal load?

Reply:
I should have been clearer with this term. Normal load in this case means cpu load but I see what you mean, normal load is ambiguous.